FFXIVClientStructs/FFXIV/Client/Network/NetworkModule.cs

```diff
@@ -0,0 +1,49 @@
+using System.ComponentModel.DataAnnotations;
+using FFXIVClientStructs.FFXIV.Application.Network;
+using FFXIVClientStructs.FFXIV.Client.System.String;
+using Unknown = nint;
+
+namespace FFXIVClientStructs.FFXIV.Client.Network;
+
+[StructLayout(LayoutKind.Explicit, Size = 0xB60)]
+public unsafe partial struct NetworkModule {
+    [FieldOffset(0x028)] public int LobbyCount;
```

Comment:
This is a byte, set at `E8 ?? ?? ?? ?? 3C 01 76 42`.
```suggestion
    [FieldOffset(0x028)] public byte LobbyCount;
```